server/.env.example

```diff
@@ -0,0 +1,2 @@
+EARTH911_BASE_URL=https://mockapi-earth911-ecohabit.up.railway.app
+EARTH911_API_KEY=SOMEKEY
```

Comment:
- can we change `SOMEKEY` to `dummykey`. the reason is, the mock api which was created for testing purposes works with the `dummykey` key. i know it's an example and all the info needed to be added when developing, but this can be in the file since it's open to everyone.
- also could you please add the below examples to the file too:

   `MONGODB_URI=`
   `JWT_SECRET=`

Reply:
@lugenx Updated the PR with the requested changes. Please review